README.md

```diff
@@ -97,6 +97,8 @@ EXAMPLES:
 
 > ℹ️ There could be multiple caches in a repo with same key. This can happen when different caches with same key have been created for different branches. it may also happen if the `version` property of the cache is different which usually means that cache with same key was created for different OS or with different [paths](https://github.com/actions/cache#inputs).
 
+## How Repo is selected
```

Comment:
Should this be put under a generic section for FAQ or troubleshooting?

Reply:
Added under  **FAQs**